sormas-api/pom.xml

```diff
@@ -2,7 +2,7 @@
 	<parent>
 		<groupId>de.symeda.sormas</groupId>
 		<artifactId>sormas-base</artifactId>
-		<version>1.59.0-SNAPSHOT</version>
+		<version>1.60.0-SNAPSHOT</version>
```

Comment:
Why did you manually change that? Rebasing or merging should be the way to go here.

Reply:
This applies to all version changes.